### PR TITLE
tests: Include the source github url as part of the mongo db issues

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The idea is to have traceability to the original error log in the mondo db issues.

So then the URL can be included in the jira issue comments when a new system fails.

### CODE REVIEW

File: tests/lib/tools/report-mongodb [hunk 1]
```diff
@@ -17,6 +17,7 @@ def _make_parser():
                         **environ_or_required('MONGO_PORT'))
     parser.add_argument('--db-name', help="Name for the database used to insert")
     parser.add_argument('--db-collection', help="Name for the collection used to insert")
+    parser.add_argument('--metadata', help="Matedata to be included in the usue")
```
Comment (bboozzoo) [inline tests/lib/tools/report-mongodb:20]:
```suggestion
    parser.add_argument('--metadata', help="Matedata to be included in the insert")
```

Comment (bboozzoo) [reply tests/lib/tools/report-mongodb:20]: does this argument as well as --db-name and --db-collect need to be marked as required?

Comment (sergiocazzolato) [reply tests/lib/tools/report-mongodb:20]: @bboozzoo those are not needed to be required because it is possible to use env vars instead

**Review by bboozzoo — APPROVED** [review]
Thanks

**Review by stolowski — APPROVED** [review]
LGTM

**Pull request merged**